compiler/rustc_codegen_ssa/src/back/linker.rs

```diff
@@ -834,6 +835,38 @@ pub struct MsvcLinker<'a> {
     sess: &'a Session,
 }
 
+impl MsvcLinker<'_> {
+    // FIXME this duplicates rustc_metadata::find_native_static_library,
+    // as the Meson/MinGW suffix for import libraries can differ
+    fn find_native_dynamic_library(name: &str, verbatim: bool, sess: &Session) -> OsString {
```

Comment:
Could you put this logic into `native_libs.rs` next to `fn try_find_native_static_library` (and generally mirror that function)?